compiler/qsc_eval/src/state.rs

```diff
@@ -483,7 +483,10 @@ fn write_latex_for_real_number(latex: &mut String, number: &RealNumber, render_o
 /// 1 is only rendered if ``render_one`` is true.
 fn write_latex_for_decimal_number(latex: &mut String, number: &DecimalNumber, render_one: bool) {
     if render_one || is_significant(number.value - 1.0) {
-        write!(latex, "{}", number.value).expect("Expected to write decimal value.");
+        /// Using round() instead of neater string formatting({:.4})
+        /// because we do not want trailing zeros (we need 0.5 and not 0.5000)
```

Comment:
```suggestion
        // Using round() instead of neater string formatting({:.4})
        // because we do not want trailing zeros (we need 0.5 and not 0.5000)
```